packages/docs/fluent-editor/demos/file-upload-handle.vue

```
@@ -0,0 +1,49 @@
+  import('@opentiny/fluent-editor').then(({ default: FluentEditor }) => {
+    if (!editorRef.value) return
+    const Delta = FluentEditor.import('delta')
+    editor = new FluentEditor(editorRef.value, {
+      theme: 'snow',
+      modules: {
+        toolbar: TOOLBAR_CONFIG,
+        uploader: {
+          // only allow image
+          mimetypes: ['image/*'],
+          handler(range: Range, files: File[]) {
+            console.log(files)
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)